lib/cs_guide/resources/drink.ex

```diff
@@ -11,7 +11,7 @@ defmodule CsGuide.Resources.Drink do
     field(:abv, :float)
     field(:entry_id, :string)
     field(:deleted, :boolean, default: false)
-    field(:description, :string)
+    field(:description, :string, default: "")
```

Comment:
@Danwhy are you still planning on adding the `Fields.description` with XSS protection?